src/app.js

```diff
@@ -1,16 +1,25 @@
 import express from 'express'
 import { Dynamo } from './dynamo.js'
 import { heartbeatInterval } from './constants.js'
+import { createServer } from 'http'
+import { Server } from 'socket.io'
 
 const port = process.env.PORT || 8086
 
 const app = express()
 app.use(express.json())
+const httpServer = createServer(app)
+const io = new Server(httpServer)
 
 const dynamo = new Dynamo()
 
 setInterval(async () => await dynamo.deleteStaleUsers(), heartbeatInterval)
 
+io.of('/game').on('connection', (socket) => {
+  console.log('Connected with client')
```

Comment:
Can we also print which client this is?

Reply:
Do you mean socket.id?

Reply:
If we have more clients I imagine we'd like to know which client connected exactly to distinguish them. Respectively clients should have if/name

Is this easy to achieve here? If not we have to add backlog item for this.

Reply:
Actually, it is just a setup next step is to use rooms(socket.io rooms) where I think it would be necessary to have this information. Or maybe now? 🤔

Reply:
Nope. That's fine. Lean agile means small steps :)